src/components/modules/DetailedPage/ProductHeader/StyledProductHeader.ts

```diff
@@ -34,12 +34,21 @@ export const Column = styled.div<{ bgColor?: string; w?: string; fa?: string }>`
   flex-direction: column;
   border-radius: 25px 25px 0px 0px;
   padding: 1rem;
+  @media (max-width: 768px) {
+    margin: 0.5rem;
+    width: 70%;
+    border-radius: 25px 25px 20px 20px;
+  }
 `;
 
 export const Row = styled.div`
   display: flex;
   flex-direction: row;
   justify-content: space-between;
+  @media (max-width: 768px) {
+    flex-direction: column;
+    align-items: center;
+  }
```

Comment:
flex-wrap might be a better solution here, 

also 
![image](https://user-images.githubusercontent.com/27841994/167436589-a6a24844-57b9-4813-8801-b77268eb48ef.png)

the tags are stacked and that's not how it looks like in the design